ui/components/app/edit-gas-fee-popover/network-status/index.scss

```diff
@@ -0,0 +1,41 @@
+.network-status {
+  margin: 24px 0 12px;
+
+  &__info {
+    border-top: 1px solid $ui-2;
+    border-bottom: 1px solid $ui-2;
+    height: 56px;
+    display: flex;
+    align-items: center;
```

Comment:
It looks these three columns get shoved to the left and don't take seem to take up the full width (this shows up when base fee is only three digits). What if you add `justify-content: space-between` here?

Reply:
That was broken coz of some changes I did in PR, I fixed it.